cpuid.go

```diff
@@ -273,6 +273,10 @@ const (
 	XSAVEOPT                             // XSAVEOPT available
 	XSAVES                               // Supports XSAVES/XRSTORS and IA32_XSS
 
+	// x86, ARM features:
+	SM3 // SM3 instructions
+	SM4 // SM4 instructions
```

Comment:
I am not really a fan of sharing features across platforms, since it can cause some unintended assumptions, like `if SM3` then I am on an ARM platform.

Let's name it `SM3_X86` or similar if you have a better suggestion.

Reply:
@klauspost
Done. Thanks a lot for the help!